### ISSUE TITLE
Button_Colors customization doesn't work with label

### ISSUE THREAD

**Issue by KillianLeroux** [egeloen/IvoryCKEditorBundle#204]
Hi,

When I try to customize colors selection with specific labels, these labels are ignored :/

Configuration in Yaml:
```yml 
colorButton_colors: FontColor1/FF9900,FontColor2/0066CC,FontColor3/F00
```

Result:
![Result colors custom](https://cloud.githubusercontent.com/assets/13987239/12199430/6a1e2f6a-b618-11e5-820a-e075c03d24d6.jpg)

Thanks for your help.

Best regards,
Killian LEROUX

**Comment by egeloen** [comment]
Can you debug what has been resolved in the config manager?

**Comment by KillianLeroux** [comment]
I'm sorry but I don't understand... ?

**Comment by egeloen** [comment]
No problem :) Basically, your configuration are normalized/injected into a service I have called the config manager and describe by the `Ivory\CKEditorBundle\Model\ConfigManager` class. I would like you debug the content of the `$configs` variable of this class.

To do so, just add a `var_dump($this->configs);die;` [here](https://github.com/egeloen/IvoryCKEditorBundle/blob/master/Model/ConfigManager.php#L106) and copy/paste the output here :)

**Comment by KillianLeroux** [comment]
Oh okay scuse me :)

Here is what you want:

``` php
array(1) {
  ["ckeditor_config"]=>
  array(13) {
    ["toolbar"]=>
    array(15) {
      [0]=>
      array(1) {
        [0]=>
        string(7) "Preview"
      }
      [1]=>
      array(1) {
        [0]=>
        string(8) "Maximize"
      }
      [2]=>
      array(7) {
        [0]=>
        string(3) "Cut"
        [1]=>
        string(4) "Copy"
        [2]=>
        string(5) "Paste"
        [3]=>
        string(9) "PasteText"
        [4]=>
        string(1) "-"
        [5]=>
        string(4) "Undo"
        [6]=>
        string(4) "Redo"
      }
      [3]=>
      array(7) {
        [0]=>
        string(4) "Find"
        [1]=>
        string(7) "Replace"
        [2]=>
        string(1) "-"
        [3]=>
        string(9) "SelectAll"
        [4]=>
        string(1) "-"
        [5]=>
        string(12) "SpellChecker"
        [6]=>
        string(5) "Scayt"
      }
      [4]=>
      string(1) "/"
      [5]=>
      array(5) {
        [0]=>
        string(4) "Bold"
        [1]=>
        string(6) "Italic"
        [2]=>
        string(9) "Underline"
        [3]=>
        string(9) "Subscript"
        [4]=>
        string(11) "Superscript"
      }
      [6]=>
      array(11) {
        [0]=>
        string(12) "NumberedList"
        [1]=>
        string(12) "BulletedList"
        [2]=>
        string(1) "-"
        [3]=>
        string(7) "Outdent"
        [4]=>
        string(6) "Indent"
        [5]=>
        string(1) "-"
        [6]=>
        string(1) "-"
        [7]=>
        string(11) "JustifyLeft"
        [8]=>
        string(13) "JustifyCenter"
        [9]=>
        string(12) "JustifyRight"
        [10]=>
        string(12) "JustifyBlock"
      }
      [7]=>
      array(3) {
        [0]=>
        string(4) "Link"
        [1]=>
        string(6) "Unlink"
        [2]=>
        string(6) "Anchor"
      }
      [8]=>
      array(4) {
        [0]=>
        string(5) "Image"
        [1]=>
        string(6) "Iframe"
        [2]=>
        string(5) "Table"
        [3]=>
        string(11) "SpecialChar"
      }
      [9]=>
      string(1) "/"
      [10]=>
      array(4) {
        [0]=>
        string(6) "Styles"
        [1]=>
        string(6) "Format"
        [2]=>
        string(4) "Font"
        [3]=>
        string(8) "FontSize"
      }
      [11]=>
      array(1) {
        [0]=>
        string(9) "TextColor"
      }
      [12]=>
      array(1) {
        [0]=>
        string(12) "RemoveFormat"
      }
      [13]=>
      string(1) "/"
      [14]=>
      array(2) {
        [0]=>
        string(14) "Texte-deux-col"
        [1]=>
        string(19) "Mise-en-avant-bleue"
      }
    }
    ["uiColor"]=>
    string(6) "F9F9F9"
    ["font_names"]=>
    string(160) "Arial/Arial, sans-serif; Dinweb/"Dinweb", Arial, sans-serif; Knockout/"Knockout 26 A", "Knockout 26 B", Arial, sans-serif; Freight/"Freight", Arial, sans-serif
"
    ["fontSize_sizes"]=>
    string(57) "Petit/small;Normal/medium;Grand/large;Très grand/x-large"
    ["colorButton_colors"]=>
    string(82) "Noir/000,Bleu clair/0078C8,Bleu foncé/1758B7,Gris foncé/2B2B2B,Gris clair/717171"
    ["colorButton_enableMore"]=>
    bool(false)
    ["scayt_autoStartup"]=>
    bool(true)
    ["scayt_sLang"]=>
    string(5) "fr_FR"
    ["stylesSet"]=>
    string(15) "styles_contenus"
    ["contentsCss"]=>
    array(2) {
      [0]=>
      string(15) "/css/styles.css"
      [1]=>
      string(42) "vendor/admin-lte/dist/css/AdminLTE.min.css"
    }
    ["extraPlugins"]=>
    string(86) "widget,lineutils,texte_deux_col,mise_en_avant_bleue,dialogui,dialog,fakeobjects,iframe"
    ["filebrowserBrowseRoute"]=>
    string(8) "elfinder"
    ["filebrowserBrowseRouteParameters"]=>
    array(0) {
    }
  }
}
```

**Comment by egeloen** [comment]
Thanks for the feedback. According to your dump, the configuration seems well propagated into the manager... I will need to investigate what's going on...

**Comment by KillianLeroux** [comment]
Okay, thanks for your investigation ;)

Do not hesitate to ask me if you need!

**Comment by egeloen** [comment]
So, after debugging it, I have found a bug in CKEditor I need to patch. Nothing wrong in the bundle :)

**Comment by egeloen** [comment]
Closing this issue and please, follow the referenced PR for your issue.

**Comment by mlewand** [comment]
The issue is going to be fixed in CKEditor 4.10.1 (see ckeditor/ckeditor-dev#2271). Thanks for letting us know!
